src/test/components/MenuButtons.test.js

```diff
@@ -63,8 +63,20 @@ const { UploadAbortedError } = jest.requireActual(
 import sha1 from '../../utils/sha1';
 jest.mock('../../utils/sha1');
 
-// Mocking compress
-jest.mock('../../utils/gz');
+// We want this module to have mocks so that we can change the return values in
+// tests. But in beforeEach below, we return the real implementation by default.
+// Note that we can't do it using the factory function because of this Jest issue:
+// https://github.com/facebook/jest/issues/14080
+jest.mock('firefox-profiler/utils/gz');
+
+beforeEach(() => {
+  const realModule = jest.requireActual('firefox-profiler/utils/gz');
+  const { compress, decompress } = require('firefox-profiler/utils/gz');
+  // $FlowExpectError Flow doesn't know about Jest mocks.
+  compress.mockImplementation(realModule.compress);
+  // $FlowExpectError Flow doesn't know about Jest mocks.
+  decompress.mockImplementation(realModule.decompress);
+});
```

Comment:
This is somewhat complex but I didn't find a better solution. See the jest issue for more information.